packages/bbui/src/Form/Core/Dropzone.svelte

```diff
@@ -159,8 +159,10 @@
           {#if selectedImage.size}
             <div class="filesize">
               {#if selectedImage.size <= BYTES_IN_MB}
-                {`${selectedImage.size / BYTES_IN_KB} KB`}
-              {:else}{`${selectedImage.size / BYTES_IN_MB} MB`}{/if}
+                {`${(selectedImage.size / BYTES_IN_KB).toFixed(1)} KB`}
+              {:else}{`${(selectedImage.size / BYTES_IN_MB).toFixed(
+                  1
+                )} MB`}{/if}
```

Comment:
While I'm in the area, and looking at the attachment UI a lot, I thought I'd truncate the file size to 1 decimal place. It wasn't truncating at all before. Let me know if you'd prefer this in a separate PR.